src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -305,6 +311,40 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
+        internal async Task HandleStartLoginMigration(
+            StartLoginMigrationParams parameters,   
+            RequestContext<StartLoginMigrationResults> requestContext)
+        {
+            try
+            {
+                // TODO AKMA: Do we need aad domain name and sql cred? If so, i'll need to set up login options as a param
```

Comment:
Regarding the comment, yes we do need AAD domain name because that is what is used for Windows logins migrations. We probably don't need Sql creds.

Reply:
So just to elaborate a bit more, if the table list of logins on the UI has Windows logins and user wants to migrate those Windows logins, then we need to have a provision to ask user for AAD domain name so that we can do the Windows user to AAD user conversion in the backend. I am not sure how in terms of UI this will translate but just wanted to let you know the use case for AAd domain name.

Reply:
Added the AAD domain name as a param and I pass it into the LoginMigrations construcor now. The UI will control whether the param is "" or has a value (for windows auth case)